types/to-regex/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "forceConsistentCasingInFileNames": true,
+        "noEmit": true,
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "lib": ["es6"]
+    },
+    "files": [
+        "index.d.ts",
+        "to-regex-tests.ts"
+    ]
+}
```

```suggestion
}

```

Add a newline at the end of the file